operations/windows-cell.yml

```diff
@@ -51,6 +52,11 @@
               ca_cert: "((diego_bbs_client.ca))"
               client_cert: "((diego_bbs_client.certificate))"
               client_key: "((diego_bbs_client.private_key))"
+            consul:
+              ca_cert: "((consul_agent.ca))"
```

Comment:
@jaresty Your suggestion about renaming `ca_cert` would require changes to the release -- not something that makes sense to do.

As far as using `consul_agent.ca` vs `diego_consul_client.ca`, I think it's fine to keep as `consul_agent.ca` for now.

Let's go ahead an merge this is in now.